src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// Write code here
+/* eslint-disable */
+import React from 'react';
+import '../../App.scss';
```

Comment:
you import a blank file? why? 🤔 
![App scss](https://user-images.githubusercontent.com/39308717/172109768-a1f0b2d3-c115-452e-ac06-9d4991c48f3e.png)